website_odoo_debranding/templates/disable_odoo.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <!-- Remove "Create a free website with Odoo" -->
+    <template
+        id="layout_footer_copyright"
+        inherit_id="website.brand_promotion"
+        customize_show="True"
```

Comment:
It isn't customizable indeed, isn't it?

Reply:
This is needed by multi-website and it's good to be able to disable this on tests. @Tardo have you tried about disabling this by default and enabling it on a hook only when no testing?

Reply:
Ok. It wasn't working in my tests, and these readme lines seem to assert a change of behavior: https://github.com/OCA/website/pull/708/commits/8f6b731dd3e67fa419e9c5c9500c5bfcadee92b6#diff-9a72aa25a2cad81905ab4f260db75c13L2-L3